.travis.yml

```diff
@@ -1,8 +1,13 @@
 language: clojure
+services:
+    - redis-server
 script:
     - lein test
     - lein yagni
     - lein eastwood
     - lein cloverage
+    - lein ring server-headless &
+    - sleep 10
```

Comment:
There's really no way to avoid this???

Reply:
There probably is, I was just getting annoyed that it wasn't working :joy: